### ISSUE TITLE
Enhance the notification email asking for api access acceptance/denial

### ISSUE THREAD

**Issue by guilieb** [MAIF/daikoku#702]
**Describe the bug**
I received an email indicating that I could accept or deny authorisation for a given API I have rights on.

The email was as follows:

```
Subject: 
Demande de clé dAPI

Body:
NAME Surname demande une clé d'api pour l'api API_NAME. Vous recevez ce message car vous êtes 
administrateur de l'équipe propriétaire.
[Accepter/rejeter la demande](https://URL/_/#/notifications)
```

This message is unclear, we don’t have the context information that have been filled by the requester, hence it’s mandatory to open the link in order to know what it’s about and/or whether it’s relevant.

My suggestions are:
1. Add the API_NAME in the outgouing email subject with maybe the identity of the requester.
2. Add the context information in the body of the email (usage of the API, comments, etc).
3. Add two links to automatically accept or deny the request directly from the email.

**Comment by quentinovega** [comment]
hi @guilieb,
an issue has been already closed ith this subject (#674) and another is in the backlog (#683).

thanks for the feedback

**Comment by guilieb** [comment]
You’re sure that it’s a duplicate ? The title of the linked issue is somehow mysterious and without any description 😶‍🌫️.
